setup.cfg

```diff
@@ -56,7 +56,7 @@ python_requires = >=3.7
 include_package_data = True
 install_requires =
     metomi-rose==2.3.*
-    cylc-flow==8.3.*
+    cylc-flow>=8.3.3.dev,<8.4
```

Comment:
Are we going to release 8.4 without the fix? 
Do we need the upper pin?
Do we want to put the .dev in here, or should we use `8.3.4`?

Reply:
> Do we want to put the .dev in here, or should we use `8.3.4`?

See latest commit

> Do we need the upper pin?

Yes, for consistency with previous `8.3.*` pin